client/lib/mixins/i18n/index.js

```diff
@@ -14,7 +14,7 @@ var debug = require( 'debug' )( 'calypso:i18n' ),
 var config = require( 'config' ),
 	numberFormatPHPJS = require( './number-format' ),
 	emitter = require( 'lib/mixins/emitter' ),
-	interpolateComponents = require( 'lib/interpolate-components' );
+	interpolateComponents = require( 'lib/interpolate-components/src/' );
```

Comment:
After investigating this would require the module to be built on its own.  The package.json already has a main property that is set to lib/index.js which is created once it is compiled.  By providing th full path the component will be compiled as part of the parent projects build process.  Do you you see an alternative?